stable/storageos-operator/templates/NOTES.txt

```diff
@@ -33,5 +33,3 @@ metadata:
 spec:
   secretRefName: storageos-api
   secretRefNamespace: default
```

Comment:
We started recommending and making all the CR examples to put the Secret in the `storageos-operator` namespace, instead of default (except for OpenShift).

Reply:
Since Helm does all the work creating the Secret, etc, I don't think it would be a problem to leave it as it is. Yet we try to keep a standard way of installing for everyones' sanity

Reply:
yes, it shouldn't be default anymore.